cjworkbench/i18n/catalogs/merge.py

```diff
@@ -12,6 +11,10 @@
 from os import remove
 from shutil import copyfile
 
+
+TEMP_BACKUP_CATALOG_FILENAME = "old.po"
```

Comment:
We can't extract to a new file. Lingui version 2 can only extract to the file it will read from when compiling.
During extraction, we delete some messages in `messages.po` by lingui's mistake. It only keeps messages that are in the js code, so the messages in `messages.po` extracted py `pybabel` will be deleted (or commented out, which is almost equally bad, since `msgctxt` gets lost if it exists).
`MESSAGES_LINGUIJS_DELETES` may be a better name, but in fact we keep all messages in `old.po`.

Reply:
Will this work:

1. Copy `messages.po` => `inprogress-messages.po`
2. Run LinguiJS extract with `inprogress-messages.po`
3. ... do more processing
4 ... move `inprogress-messages.po` => `messages.po`

Reply:
In order for LinguiJS to extract to `inprogress-messages.po`, we must do one of the two following things:
1. Alter its configuration file dynamically, either on extract or on compile
2. Keep the file `inprogress-messages.po` as the final one in step 4, which of course is equivalent to the current procedure

Reply:
Sigh, that's frustrating. Okay. Thank you for guiding me through this thought experiment.

`MID_EXTRACT_CATALOG_FILENAME`?